cob_command_gui/package.xml

```diff
@@ -13,6 +17,8 @@
   <author email="fmw@ipa.fhg.de">Florian Weisshardt</author>
 
   <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_PYTHON_VERSION == 2">python-setuptools</buildtool_depend>
+  <buildtool_depend condition="$ROS_PYTHON_VERSION == 3">python3-setuptools</buildtool_depend>
 
   <exec_depend>cob_msgs</exec_depend>
   <exec_depend>cob_script_server</exec_depend>
```

Comment:
we'll have to decide what to do with this:
`python-gtk2` is no longer available for focal - and there is nothing to easily convert this to

```
E: Unable to locate package python-gtk2
ERROR: the following rosdeps failed to install
  apt: command [apt-get install -y -qq python-gtk2] failed
```

Reply:
I'm going to do a partial release with the packages that pass `noetic` without major changes....and open an issue for `cob_command_gui`